change path of css stylesheet in home.hbs

shreyag12 opened the pull request:
Updated path of style.css in home.hbs

littlestar642 commented:
There is no need to change the directory of the css files as in my console i dont see any error. You need to rebase your branch and set up the project correctly.

shreyag12 commented:
No I did not change the directory of the css file, I just changed it to public/css/style.css becuase in home.hbs it is css/style.css. Because it was not working after i rebased the code.
